neptune/new/sync.py

```diff
@@ -147,7 +148,11 @@ def get_offline_runs_ids(base_path: Path) -> List[str]:
 def partition_runs(base_path: Path) -> Tuple[List[ApiRun], List[ApiRun], int]:
     synced_runs_ids = []
     unsynced_runs_ids = []
-    for run_path in (base_path / ASYNC_DIRECTORY).iterdir():
+    async_path = base_path / ASYNC_DIRECTORY
+    if not async_path.is_dir():
```

Comment:
What is the purpose of it?

Reply:
If we make `offline` run only, there will be `./neptune/offline`, but no `./neptune/assync` dir, so command would fail.
I've treated missing `async` dir as empty one.